R/uvhydrograph-data.R

```diff
@@ -307,9 +259,9 @@ getInverted <- function(data, renderName, plotName) {
     
   } else if (plotName == "secondary") {   
     dataName <- switch(renderName,
-                       corr_UV2 = "downsampledSecondarySeries",
-                       est_UV2 = "downsampledSecondarySeries",
-                       uncorr_UV2 = "downsampledSecondarySeriesRaw")
```

Comment:
Yes, but unfortunately the java side does not attached inverted flags to the secondary downsampled stuff. I did not update the java to do that because I think we might (later) remove the inverted flags from the downsampled series all together. It's wasted processing on the backend if we continue to include the full resolution version.